modules/ossm-vs-istio.adoc

```diff
@@ -90,7 +90,30 @@ spec:
 
 {ProductName} includes CNI plug-in, which provides you with an alternate way to configure application pod networking. The CNI plug-in replaces the `init-container` network configuration eliminating the need to grant service accounts and projects access to Security Context Constraints (SCCs) with elevated privileges.
 
+The Istio CNI plugin is enabled through Multus CNI. The Istio operator creates a
```

Comment:
@luksa Can you take a look here? I took this from your PR in https://issues.redhat.com/browse/MAISTRA-1611. And hopefully also works with https://issues.redhat.com/browse/MAISTRA-1665.

Reply:
LGTM